src/old-components/Guilds/ActionsBuilder/SupportedActions/ERC20Transfer/ERC20TransferInfoLine.tsx

```diff
@@ -9,7 +9,10 @@ import { useMemo } from 'react';
 import { FiArrowRight, FiNavigation } from 'react-icons/fi';
 import { MAINNET_ID, shortenAddress } from 'utils';
 
-const ERC20TransferInfoLine: React.FC<ActionViewProps> = ({ decodedCall }) => {
+const ERC20TransferInfoLine: React.FC<ActionViewProps> = ({
+  decodedCall,
+  compact,

```

Comment:
Do you think it makes sense to have a different component altogether for the compact version? A `CompactInfoLine` component?

Reply:
For now, there are just small changes, like hiding some text or changing sizes, and seems ok to me for now, but let me know what you think.